src/scylla/db.clj

```diff
@@ -330,7 +330,10 @@
   "Returns a comma-separated string of seed nodes for boostrap.
   Return all nodes to speed up bootstrap."
   [test]
-  (->> (:nodes test)
+  (->> test
+       :nodes
+       first
+       list
```

Comment:
This is super weird, you're taking the first element of `(:nodes test)`, then making a single-element list of it, then resolving the address and `join`ing the list with `"`...

How about
```
(->> test :nodes first dns-resolve str)
```

Also, the doc string needs to be updated.

Reply:
Also, piping `test` to `:nodes` using `->>` is kinda awkward, I think you should stick to `(:nodes test)`.

The `->>` macro is good for expressing a sequence of transformations performed on some data structure. But `:nodes` is not a "transformation" performed on `test` - well formally it is, but it's weird for the reader of the code to look at this in this manner.

Reply:
Done.